webpack/JobInvocationDetail/JobInvocationOverview.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import DefaultLoaderEmptyState from 'foremanReact/components/HostDetails/DetailsCard/DefaultLoaderEmptyState';
+import { translate as __, documentLocale } from 'foremanReact/common/I18n';
+
+const JobInvocationOverview = ({ data }) => {
+  const {
+    ssh,
+    start_at: runOn,
+    ssh_user: sshUser,
+    template_invocations: template,
+  } = data;
+  const effectiveUser = ssh ? ssh.effective_user : null;
+  const templateName = template ? template[0].template_name : null;
+
+  const dateOptions = {
+    day: 'numeric',
+    month: 'short',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    hour12: false,
+    timeZoneName: 'short',
+  };
+  const dateLocaleFormatted = new Date(runOn)
+    .toLocaleString(documentLocale(), dateOptions)
+    .replace(/(\d{4}) /, '$1, ');
+
+  return (
+    <DescriptionList
+      columnModifier={{
+        default: '2Col',
+      }}
+      isHorizontal
+      isCompact
+      isFluid
+      isAutoColumnWidths
+    >
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Effective user:')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {effectiveUser || <DefaultLoaderEmptyState />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Run on:')}</DescriptionListTerm>
```

Comment:
Probably should change the text according to if it already ran or if its going to run in the future, can be done in the next pr but would be nice to have some todo note

Reply:
![image](https://github.com/theforeman/foreman_remote_execution/assets/30431079/6926ef74-3760-4f3f-a73f-a4dd00b2f0f5)